### PR TITLE
Add instructions for playing with qlora model to colab example

### PR DESCRIPTION
**Pull request by jaredpalmer**
# Description

Add instructions for launching a gradio app on google colab to play with finetuned model

## Screenshots (if appropriate)
CleanShot 2024-02-13 at 14 43 57@2x

### CODE REVIEW

**Review by winglian — APPROVED** [review]

**Comment by JohanWork** [comment]
For me this example dosen't work on the free tier. Getting 'RuntimeError: FlashAttention only supports Ampere GPUs or newer.' Would be nice to have a setup that work for everyone without having to have a premium account :) Changing to 
```python 
!accelerate launch -m axolotl.cli.inference /content/test_axolotl.yaml \
    --qlora_model_dir="./qlora-out" --gradio
```
for example work for me. 
@jaredpalmer 
For reference I am using a T4

**Comment by NanoCode012** [comment]
I think @JohanWork 's suggestion is nice. It makes sense also to reuse the same config from training instead of another yaml.

File: examples/colab-notebooks/colab-axolotl-example.ipynb [hunk 1]
```diff
@@ -177,6 +177,24 @@
...
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# Buy using the ! the comand will be executed as a bash command\n",
+        "!accelerate launch -m axolotl.cli.inference src/axolotl/examples/tiny-llama/qlora.yml \\\n",
```
Comment (JohanWork) [inline examples/colab-notebooks/colab-axolotl-example.ipynb]:
```suggestion
        "!accelerate launch -m axolotl.cli.inference /content/test_axolotl.yaml \\\n",
```

Comment (JohanWork) [reply examples/colab-notebooks/colab-axolotl-example.ipynb]: @winglian suggest to change the config file before merging, see comments above.

Comment (JohanWork) [reply examples/colab-notebooks/colab-axolotl-example.ipynb]: @NanoCode012 can you merge this? :)

**Pull request merged**